Matchismo/Matchismo/Model/Card.h

```diff
@@ -14,7 +14,7 @@ struct MatchResult {
   NSMutableArray<Card *> *matches;
 };
 
-@property (strong, nonatomic) NSString *contents;
+@property (strong, nonatomic) NSAttributedString *contents;
```

Comment:
Bit odd that we're using an attributed string in what's presumably a model object?
Better just to keep "business" logic here and put the UX-related logic (i.e. choice of fonts, colours or whatever) in the UI layer.